signale.js

```diff
@@ -192,7 +192,7 @@ class Signale {
       }
     }
 
-    if (msg instanceof Error) {
+    if (msg instanceof Error && typeof msg.stack !== 'undefined') {
```

Comment:
Nice approach! We could also rely upon type conversion and prevent any falsy value from making it through, that way rare cases where `stack` is set, for example, to `null` are too covered;

```js
if (msg instanceof Error && msg.stack) {
```